src/NzbDrone.Core/Notifications/Signal/SignalPayload.cs

```diff
@@ -0,0 +1,11 @@
+namespace NzbDrone.Core.Notifications.Signal
+{
+    public class SignalPayload
+    {
+        public string Message { get; set; }
+
```

Comment:
Nit: no need for new lines between these simple props.

Reply:
Hello, i changed it in my last commit : is everything ok for you ?

Reply:
Yeah, that's great, thanks